tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -742,6 +737,9 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
     if 'help' in args and args['help']:
         show_mo_convert_help()
         return None, None
+    if 'version' in args and args['version']:
+        print('Version of Model Optimizer is: {}'.format(get_version()))
+        return None, None
```

Comment:
why do we need it?

Reply:
In cli pipeline version is shown using special action in cli_parser. So actual showing of version is in parse_args().

In Python API path cli_parser actions are executed only for string parameter. For 'input_model' for example we should not check path if model came from memory and it is not string. 

For this reason 'version' param processing was added here, because otherwise for in Python API it was ignored.

But it is probably should not be checked here. I added execution actions of bool params as well, so it is not needed here anymore.